common/php/class-module.php

```diff
@@ -665,8 +665,8 @@ public function users_select_form($selected = null, $args = null)
              * @param array $roles
              * @param int   $post_id
              */
-            $roles = apply_filters('publishpress_notification_roles_meta_box', get_editable_roles(),
-                (int)$_GET['post']);
+            $post_id = isset($_GET['post']) ? (int)$_GET['post'] : null;
```

Comment:
There's no need to reassign `$post_id` because on this line you're doing exactly the same as on line `645`. We can simply remove this line and reuse the variable instead.

Reply:
I've removed the offending line now.